core/src/main/java/io/lionweb/lioncore/java/model/ClassifierInstance.java

```diff
@@ -1,14 +1,36 @@
 package io.lionweb.lioncore.java.model;
 
+import io.lionweb.lioncore.java.language.Annotation;
 import io.lionweb.lioncore.java.language.Classifier;
 import io.lionweb.lioncore.java.model.impl.ProxyNode;
 import java.util.Collection;
 import java.util.List;
+import javax.annotation.Nonnull;
 
 public interface ClassifierInstance<T extends Classifier<T>> extends HasFeatureValues {
   /** Return all the annotations associated to this ClassifierInstance. */
+  @Nonnull
   List<AnnotationInstance> getAnnotations();
 
+  /**
+   * Given a specific Annotation type it returns either the list of instances of that Annotation
+   * associated to the Node.
+   */
+  @Nonnull
+  List<AnnotationInstance> getAnnotations(@Nonnull Annotation annotation);
+
+  /**
+   * If an annotation instance was already associated under the Annotation link used by this
```

Comment:
I thought we removed `Annotation.multiple` from M3? So we can always attach several instances of the same annotation.

Reply:
You are right. The logic was already like you suggested. I have now updated also the comment.